src/App.tsx

```diff
@@ -1,24 +1,187 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+// eslint-disable-next-line object-curly-newline
```

Comment:
Please remove the unnecessary eslint-disable-next-line rule for 'object-curly-newline' as it is not required in this file.